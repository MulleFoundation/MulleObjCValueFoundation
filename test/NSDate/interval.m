#ifndef __MULLE_OBJC__
# import <Foundation/Foundation.h>
#else
# import <MulleObjCValueFoundation/MulleObjCValueFoundation.h>
#endif


int   main( void)
{
   NSDate   *value;

   value = [[[NSDate alloc] initWithTimeIntervalSinceReferenceDate:0] autorelease];
   printf( "2001: %.1f\n", [value timeIntervalSinceReferenceDate]);
   printf( "1970: %.1f\n", [value timeIntervalSince1970]);

   value = [[[NSDate alloc] initWithTimeIntervalSinceReferenceDate:1.5] autorelease];

   printf( "2001: %.1f\n", [value timeIntervalSinceReferenceDate]);
   printf( "1970: %.1f\n", [value timeIntervalSince1970]);

   return( 0);
}
