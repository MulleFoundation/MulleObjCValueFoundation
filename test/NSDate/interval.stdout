2001: 0.0
1970: 978307200.0
2001: 1.5
1970: 978307201.5
